app/code/Magento/Widget/Model/ResourceModel/Widget/Instance.php

```diff
@@ -29,10 +31,10 @@ protected function _construct()
     /**
      * Perform actions after object load
      *
-     * @param \Magento\Widget\Model\Widget\Instance $object
```

Comment:
I'd prefer having as precise as possible annotation, is this method used for processing any abstract models except `Magento\Widget\Model\Widget\Instance` ?